.github/workflows/ci-formatting-and-dependencies.yml

```diff
@@ -25,16 +25,29 @@ jobs:
           black --check .
           npx prettier --check envision/web/src
 
-  check-requirements-change-linux:
+  update-requirements-linux:
     runs-on: ubuntu-18.04
     if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name != github.repository
     container: huaweinoah/smarts:v0.4.18-minimal
     steps:
+      - name: Install git
+        run: |
+          add-apt-repository -y ppa:git-core/ppa && \
+            apt-get update && \
+            apt-get install -y git
       - name: Checkout
         uses: actions/checkout@v2
-      - name: Test pip packages
+      - name: Update requirements
         run: |
           cd $GITHUB_WORKSPACE
-          . ./utils/setup/test_pip_packages.sh
```

Comment:
It looks like this script is no longer called anywhere. Should it be removed or updated?

Reply:
The `/utils/setup/test_pip_packages.sh` file is removed.